projects/core/src/smart-edit/services/smart-edit.service.ts

```diff
@@ -17,8 +17,8 @@ export class SmartEditService {
   private _currentPageId: string;
 
   constructor(
-    private cmsService: CmsService,
-    private routingService: RoutingService,
+    protected cmsService: CmsService,
+    protected routingService: RoutingService,
     private zone: NgZone,
     winRef: WindowRef
```

Comment:
Shouldn't this be private?

Reply:
winRef is not `public` or `private`. It is only used in `constructor`. Without access modifier, the constructor parameter is nothing more than a method parameter.